salesforce/models.py

```diff
@@ -23,16 +23,12 @@ def __str__(self):
 
 class AdoptionOpportunityRecord(models.Model):
     opportunity_id = models.CharField(max_length=255, unique=True)
-    account_id = models.CharField(max_length=255, null=True, blank=True) # TODO: for deletion after switching to UUID
     account_uuid = models.UUIDField(null=True)
```

Comment:
wondering if it might help data consistency to have the contact id here too - since we do have that in the /api/user endpoint